### DISCUSSION TITLE
when i use `$fetch` in /server/api/**.ts, the backend server, like java can not got the referer/origin from headers, how to fix or set it?

### DISCUSSION THREAD

**Discussion by JiatLn** [nuxt/nuxt#16272] [Questions]
when i use `$fetch` in /server/api/**.ts, the backend server(such as java) can not got the `referer/origin` from headers,

**Comment by TheAlexLichter** [comment]
You could pass it through an extra header, e.g. `X-Referer`, ...

**Reply by JiatLn** [reply]
that is a way, but i want to know other way to fix it without set other extra headers

**Reply by TheAlexLichter** [reply]
I don't think you can "forward" them right now. Feel free to create a feature request for that though ☺️
